README.md

```diff
@@ -25,6 +25,8 @@ rails server
 cp .env.example .env # altere as chaves caso queira trocar a aplicação
 docker-compose build
 docker-compose run --rm web rake db:setup
+docker-compose run --rm web rake db:migrate
```

Comment:
I received error too... |:

How we use rails 5... Do you think about change to only one command?

```
docker-compose run --rm web rails db:create db:migrate db:seed
```